source/steamshim/steamshim_child.h

```diff
@@ -26,6 +28,7 @@ typedef struct STEAMSHIM_Event
     int okay;
     int ivalue;
     float fvalue;
+    long long lvalue;
```

Comment:
Why not re-use this for both?
The bigger this struct gets, the more 'useless' date you are flinging between processes for every command.

Reply:
https://github.com/CoolElectronics/warfork-qfusion/commit/7efe098a04ab0ebd96b3bd34cccb12be3e83ebcc